resources/assets/js/alerts.js

```diff
@@ -86,16 +98,9 @@ function handleClick(e) {
 
 const indexWrapper = document.querySelector(".manager-poster-index");
 // Double check we're on the manager index page, grab all the
-// 'Send for Review' and 'Delete' buttons, and register the click event handler
+// 'Delete' buttons, and register the click event handler
 // defined above.
 if (typeof indexWrapper !== "undefined" && indexWrapper != null) {
-  const reviewButtons = Array.from(
```

Comment:
We don't use this anymore actually, the send for review button has been moved to the Review step in the JPB.

Reply:
![Screen Shot 2020-02-13 at 12 04 48 PM](https://user-images.githubusercontent.com/6796588/74453667-133c5b80-4e59-11ea-91ee-b2233cc25b18.png)
This is on the manager index view of the current branch

Reply:
So the Send for Review button is now a link, that takes you the Review step in the JPB. It used to be a button that opened the sweet alert modal. That's the content that I removed here since were not using it anymore.